client.go

```diff
@@ -11,6 +11,8 @@ import (
 	"go.mongodb.org/atlas/mongodbatlas"
 )
 
+const userAgentPluginName = "secrets-mongodbatlas"
```

Comment:
I'm trying to establish a `<type>-<name>` pattern for the plugin name used in the user-agent comment section. This is the reverse of how this was before this PR. I will check to see if this is a breaking change and reverse the order back if so. I could imagine this being parsed for usage attribution on the MongoDB atlas side, but I don't know if anyone is doing that for certain.

Reply:
I changed this back to the original order. They're tracking usage attribution via the user agent. The ordering isn't worth the cost of coordination and change involved for them to update tracking.